apps/omg/test/omg/integration/root_chain_coordinator_test.exs

```diff
@@ -39,34 +39,4 @@ defmodule OMG.RootChainCoordinatorTest do
 
     :ok
   end
-
-  @tag fixtures: [:alice, :db_initialized, :root_chain_contract_config]
-  test "can do a simplest sync",
```

Comment:
this test is irrelevant now. we have cabbage to prove that a deposit is synced and that it works.